CustomMission/TRGMSetDefaultMissionSetupVars.sqf

```diff
@@ -1,115 +0,0 @@
-
```

Comment:
This file is needed, see here: https://youtu.be/K6u34Ekxsmo?t=115
(have set the video to start at 1:55, as that is where i go into detail about the values)

Reply:
Slowly working it out, but see [here](https://github.com/treendy/TRGM2.Malden/pull/134/commits/d1084649ff41c0f8abaa7bcb34ad21ff150dc9d0?file-filters%5B%5D=.sqf#diff-88745d6c427a1bc1b88927173576e07aaaeafd47f84d50b2cb4d04e788663426).